src/Page/Options.elm

```diff
@@ -111,126 +111,64 @@ update navKey msg model =
 view : Model -> Html Msg
 view model =
     Search.view { toRoute = Route.Options, categoryName = "options" }
-        "Search NixOS options"
+        [ text "Search more than "
+        , strong [] [ text "10 000 options" ]
```

Comment:
Any strategies to help ensure this number doesn't drift too far?

Reply:
I think there are essentially two ways.

- dynamic - fetch this information over network
- static - inline this information during build

I think @garbas will know what is the right call more than me. However I agree that hard-coding this is won't be scalable. But I'm ok resolving this in separate issue/PR.

Reply:
Ideally this would be done as a counter that counts from 0 to a number which is fetched from the backend meanwhile. But for now this was the easier hack.

Reply:
There is a Count API that we can use from elasticsearch https://www.elastic.co/guide/en/elasticsearch/reference/current/search-count.html